examples/FilterExample.js

```diff
@@ -55,7 +55,7 @@ class FilterExample extends React.Component {
   _onFilterChange(e) {
     if (!e.target.value) {
       this.setState({
-        filteredDataList: this._dataList,
+        filteredDataList: this._dataList
```

Comment:
I think the trailing commas should be left since they will cause unnecessary git diffs in future.